reminders/plugin_bot.go

```diff
@@ -99,15 +99,19 @@ var cmds = []*commands.YAGCommand{
 		Description:         "Lists your active reminders",
 		SlashCommandEnabled: true,
 		DefaultEnabled:      true,
+		IsResponseEphemeral: true,
 		RunFunc: func(parsed *dcmd.Data) (interface{}, error) {
 			currentReminders, err := GetUserReminders(parsed.Author.ID)
 			if err != nil {
 				return nil, err
 			}
 
-			out := "Your reminders:\n"
-			out += stringReminders(currentReminders, false)
-			out += "\nRemove a reminder with `delreminder/rmreminder (id)` where id is the first number for each reminder above.\nTo clear all reminders, use `delreminder` with the `-a` switch."
+			out := "You have no reminders. Create reminders with the `Remindme` command."
+			if len(currentReminders) > 0 {
+				out = "Your reminders:\n"
+				out += stringReminders(currentReminders, false)
+				out += "\nRemove a reminder with `delreminder/rmreminder (id)` where id is the first number for each reminder above.\nTo clear all reminders, use `delreminder` with the `-a` switch."
+			}
 			return out, nil
```

Comment:
Very minor suggestion, but I would personally write
```go
if len(currentReminders) == 0 {
	return "You have no reminders. Create reminders with the `Remindme` command.", nil
}

out := "Your reminders\n"
// ...
return out, nil
```
This avoids the reassignment to `out` on line 111,
```go
out = "Your reminders:\n"
```
which confused me on first read — I wasn't sure if it was intended or a typo. (Though it's clear after reading the text, the suggested structure avoids this problem entirely.)

Up to you, though. As I said, this is most minor.

Reply:
Oh yes that does seem cleaner, sounds good